pymilvus/orm/schema.py

```diff
@@ -412,6 +412,9 @@ def __init__(self, name: str, dtype: DataType, description: str = "", **kwargs)
         self.element_type = kwargs.get("element_type")
         if "mmap_enabled" in kwargs:
             self._type_params["mmap_enabled"] = kwargs["mmap_enabled"]
+        if "enable_ngram_index" in kwargs:
+            self._type_params["min_gram"] = kwargs["min_gram"]
```

Comment:
If user passes "enable_ngram_index" and not "min_gram", "max_gram", this line will raise KeyError